nvr/nvr.py

```diff
@@ -330,20 +330,33 @@ def main(argv=sys.argv, env=os.environ):
         print_sockaddrs()
         return
 
-    address = options.servername or env.get('NVIM_LISTEN_ADDRESS') or '/tmp/nvimsocket'
+    if sys.platform == 'win32':
+        address = options.servername or env.get('NVIM_LISTEN_ADDRESS') or None
+        if address == None:
+            print("Windows workaround: could not find an address! Set environment variable 'NVIM_LISTEN_ADDRESS', or pass a server address when calling nvr: `nvr --servername \\some\\server\\address`.")
+            # possibility: what could be a good temp file location on windows? Once we decide on this, we could use the subprocess possibility described below to start an nvim process on windows using this temp server. 
```

Comment:
> what could be a good temp file location on windows?

Instead of making something Windows specific, why not make it more generic?

```python
import os.path
import tempfile
...
address = options.servername or env.get('NVIM_LISTEN_ADDRESS') or os.path.join(tempfile.gettempdir(), 'nvimsocket')
```

Reply:
That seems to be a very good idea to me, but probably something @mhinz would have to comment on.

Reply:
@jamessan have a look at the latest commit, and let me know what you think, i implemented changes based on ideas :)